smartsim/_core/entrypoints/telemetrymonitor.py

```diff
@@ -591,6 +599,7 @@ def main(
     )
 
     cooldown_duration = cooldown_duration or CONFIG.telemetry_cooldown
+    logger.warning(cooldown_duration)
```

Comment:
Can we add text here to say what the number is? 

`logger.warning(f"Cooldown period (seconds): {cooldown_duration}")`

Reply:
Yes, but demoted to `debug` level.